### PR TITLE
Remove the Register link

### PR DESCRIPTION
**Pull request by cardona6**
Remove the Register link from Admin home page

### CODE REVIEW

File: server/views/register.hbs [hunk 1]
```diff
@@ -8,7 +8,7 @@
     <label>Name</label>
     <input type="text" class="form-control" placeholder="Name" name="name">
   </div>
-  <div class="form-group">
+  <div class="form-group"></div>
```
Comment (kabaros) [inline server/views/register.hbs]: why did you change this?

Comment (cardona6) [reply server/views/register.hbs]: Fix this part. It was me wrong. and add it unintentionally

**Review by kabaros — APPROVED** [review]
